source/modules/MEI.shared.xml

```diff
@@ -2720,6 +2720,40 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.repeatMark.log" module="MEI.shared" type="atts">
+    <desc xml:lang="en">Logical domain attributes.</desc>
+    <classes>
+      <memberOf key="att.controlEvent"/>
+      <memberOf key="att.duration.additive"/>
+      <memberOf key="att.startEndId"/>
+      <memberOf key="att.timestamp2.logical"/>
+    </classes>
+    <attList>
+      <attDef ident="func" usage="req">
+        <desc xml:lang="en">Indicates the function of the mark.</desc>
+        <valList type="closed">
+          <valItem ident="coda">
+            <desc xml:lang="en">Coda (SMuFL E048 or Unicode 1D10C).</desc>
+          </valItem>
+          <valItem ident="segno">
+            <desc xml:lang="en">Segno (SMuFL E047 or Unicode 1D10B).</desc>
+          </valItem>
+          <valItem ident="dalSegno">
+            <desc xml:lang="en">Dal segno (SMuFL E045 or Unicode 1D109).</desc>
+          </valItem>
+          <valItem ident="daCapo">
+            <desc xml:lang="en">Da capo (SMuFL E046 or Unicode 1D10A).</desc>
+          </valItem>
+          <valItem ident="fine">
+            <desc xml:lang="en">Fine.</desc>
+          </valItem>
+          <valItem ident="other">
+            <desc xml:lang="en">Unspecified symbol.</desc>
```

Comment:
What is an 'other' mark? How would a user specify the actual symbol if they were to choose `other`?

Reply:
If the user specifies `other` then I think it should be required to define the symbol with the `glyph.*` attributes.

Reply:
Good point. Also the attribute name is `func`, not `form`. Are there more possible functions we don't know about yet? Then it would be better to name this `unkown` over `other`, because the symbol could be encoded whatsoever.

Reply:
+1 to renaming it to `form` -- arguably, the function is similar, or the same, for some of these, but the form is different.

Reply:
I do not think we can bind the use of `other` with the definition of a glyph. The element can contain text because sometimes this is how a mark is described. The cases where `other` is used are probably the most likely not to be just a single symbol. For example, some thing like:
```xml
<expanMark func="other">Repeat this section as many times as needed until all the dancers have left the stage</expanMark>
```
I hope this also explains why `@func` seems preferable than `@form`.

Reply:
In that instance "other" doesn't actually bring anything, though, and it would be better to omit it entirely.